src/components/EnvironmentBlock.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h1>Try to use calculator</h1>
+    <button class="button_environment">View</button>
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+
+div {
+  --main_block_paddings_big_screen: 80px;
+  --main_block_paddings_small_screen: 20px;
+  color: #4f4f4f;
+  margin-top: 60px;
+  height: 80px;
+  background-color: #fbfbfb;
+  border-bottom: 1px solid #ebebeb;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: var(--main_block_paddings_big_screen);
+  padding-right: var(--main_block_paddings_big_screen);
+}
+@media screen and (max-width: 500px) {
+  div {
```

Comment:
Please add class for this div and style it through the class. Because if you add another div into the layout, then this styles will apply automatically to it.

Reply:
ok